spec/requests/users_spec.rb

```diff
@@ -1,7 +1,38 @@
 require 'rails_helper'
 
 RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+  describe 'GET /users#index' do
+    it 'if response status code is correct' do
```

Comment:
🚀 Well done!!! All the required tests are written and they all pass 👏🏾💯 

![image](https://github.com/Theodoraldo/blog-app/assets/38088818/84e503bd-9ec6-4dd9-b926-ad7da0911052)